CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Adjoint plugin import failures after `jax.numpy.DeviceArray` -> `jax.Array` in jax 0.4.14.
+- Changed definition of shapely box bounds in Geometry.intersections_2dbox() for issue 1039
```

Comment:
We usually don't reference issues in the changelog. Just `Fixed definition of shapely box bounds in Geometry.intersections_2dbox().` works (note fixed as opposed to changed).

Could you update that and then squash the commits into 1? Then we can merge.